### DISCUSSION TITLE
Add the ability to play peertube/invidious instance videos, just like youtube videos.

### DISCUSSION THREAD

**Discussion by Empathetic** [stoatchat/discussions#313] [Feature Suggestions]
Add the ability for the client to be able to play peertube and invidious (Youtube front-end) directly in the revolt client/web client. It's already possible with youtube videos just like Discord but adding the ability to play videos directly from peertube and invidious (Youtube front-end) instances would be awesome. Don't know how feasible this is but that would put you guys one step ahead of discord.

**Comment by UnixPhonez** [comment]
Also be sure to add odysee videos
